lang/en/activities.php

```diff
@@ -12,4 +12,5 @@
     Activity::REACTIVATED_SUBSCRIPTION->value => ':name has reactivated a subscription.',
     Activity::REFUNDED_ORDER->value => ':name has received a refund of :amount :currency.',
     Activity::RESUMED_SUBSCRIPTION->value => ':name has resumed a subscription.',
+    Activity::SKIPPED_MENU->value => ':name has skipped a menu for :subscription on :date.',
```

Comment:
What is the `on :date` for? Every feed already has the timestamp of when the action happened.

You can change the message for:

> :name has skipped the delivery of :date for :subscription.